cvat/apps/iam/rules/jobs.rego

```diff
@@ -4,7 +4,7 @@ import data.organizations
 
 # input: {
 #     "scope": <"view"|"list"|"update:state"|"update:stage"|"update:assignee""delete"|
-#         "view:annotations"|"update:annotations"|"delete:annotations"|"view:data"> or null,
+#         "view:annotations"|"update:annotations"|"delete:annotations"|"view:data"|"export:dataset"> or null,
```

Comment:
@kirill-sizov , do we need to update CSV file as well?

Reply:
@nmanovic Yes, we need, I have updated